### ISSUE TITLE
Default to creating VERSION is none exists ?

### ISSUE THREAD

**Issue by gedw99** [greenpau/versioned#2]
Basic reproduction... 

``` 

EX=$(PWD)/example
run: run-clean
	mkdir -p $(EX)

	# Must have this empty file first.
	cd $(EX) && touch VERSION

	cd $(EX) && versioned -h

	cd $(EX) && versioned -init

	cd $(EX) && versioned

run-clean:
	rm -rf $(EX)

```

**Comment by greenpau** [comment]
@gedw99 , thank you for the issue! Will address it the coming week!

**Comment by greenpau** [comment]
@gedw99 , released [v1.0.27](https://github.com/greenpau/versioned/releases/tag/v1.0.27). Thank you for the issue!
